Analysis/Tasks/PWGHF/HFD0CandidateSelector.cxx

```diff
@@ -107,42 +63,56 @@ struct HFD0CandidateSelector {
     return true;
   }
 
+  template <typename T>
+  int findPTBin(T pT)
+  {
+    if ((pT < cuts->get((int)0, "pT lo")) || (pT >= cuts->get(cuts->rows() - 1, "pT hi"))) {
+      return -1;
+    }
+    for (auto i = 0u; i < cuts->rows(); ++i) {
+      if (pT < cuts->get(i, "pT hi")) {
+        return i;
+      }
+    }
+    return -1;
+  }
```

Comment:
Since this function will look the same in all the other selectors, would it make sense to move it to `HFSelectorCuts.h` and pass the `cuts` by reference to it?

Reply:
It needs to be a method of the task class to have access to cut array. But the logic of it can indeed extracted to an external function later.

Reply:
`HFD0CandidateSelector` is a `struct`. Does that make the `Configurable`s public?

Reply:
While it is a public data member, we do not have direct access to the task object to access it - the task is managed by framework.